.github/workflows/lint.yaml

```diff
@@ -13,5 +13,7 @@ jobs:
       with:
         node-version: '16'
     - run: npm ci
+    - run: cp .env.example .env
+    - run: npx svelte-kit sync
     - run: npm run lint

```

Comment:
After adding the typecheck task, call it separately here before lint.

Reply:
done